src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -2434,5 +2438,53 @@ class Program
     int [|a|];
 }");
         }
+
+        [WorkItem(40053, "https://github.com/dotnet/roslyn/issues/40053")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestEqualsNullableAnnotation()
+        {
+            await TestWithPickMembersDialogAsync(
+@"
+#nullable enable
+using System;
+
+namespace N
+{
+    public class C[||]
+    {
+        public int X;
+    }
+}",
+@"
+#nullable enable
+using System;
+
+namespace N
+{
+    public class C
+    {
+        public int X;
+
+        public override bool Equals(object? obj)
+        {
+            return obj is C c &&
+                   X == c.X;
+        }
+
+        public static bool operator ==(C? left, C? right)
+        {
+            return global::System.Collections.Generic.EqualityComparer<global::N.C>.Default.Equals(left, right);
```

Comment:
we should not be generated `global::` here.

Reply:
This is a separate issue. @y87feng  do you mind filing? I can help you file as needed.

Reply:
not certain how it's a different issue :)  isn't this caused because of the new usage of nullable types heres?

Reply:
I don't think it is because of the new usage of nullable types:) I can check if there is an issue already

Reply:
I'm curious what woudl cause it then.  None of hte other tests exhibit htis issue.

Reply:
The behavior existed before this change. It might be related to enabling nullable/using csharp 8, but was not directly caused by this change.